snuba/datasets/processors/metrics_bucket_processor.py

```diff
@@ -117,28 +114,12 @@ def process_message(
         )
 
 
-class PolymorphicMetricsProcessor(MetricsBucketProcessor):
-    def _should_process(self, message: Mapping[str, Any]) -> bool:
-        return message["type"] in {
-            InputType.SET.value,
-            InputType.COUNTER.value,
-            InputType.DISTRIBUTION.value,
-        }
-
-    def _process_values(self, message: Mapping[str, Any]) -> Mapping[str, Any]:
-        if message["type"] == InputType.SET.value:
-            return values_for_set_message(message)
-        elif message["type"] == InputType.COUNTER.value:
-            value = message["value"]
-            assert isinstance(
-                value, (int, float)
-            ), f"{ILLEGAL_VALUE_FOR_COUNTER} {INT_FLOAT_EXPECTED}: {value}"
-            return {"metric_type": OutputType.COUNTER.value, "count_value": value}
-        else:  # message["type"] == InputType.DISTRIBUTION.value
-            return values_for_distribution_message(message)
-
-
 def timestamp_to_bucket(timestamp: datetime, interval_seconds: int) -> datetime:
     time_seconds = timestamp.timestamp()
     out_seconds = interval_seconds * (time_seconds // interval_seconds)
     return datetime.fromtimestamp(out_seconds, timestamp.tzinfo)
+
+
+class PolymorphicMetricsProcessor(RustCompatProcessor):
+    def __init__(self) -> None:
+        super().__init__("PolymorphicMetricsProcessor")
```

Comment:
Just to confirm - this change is just for development so we can run all of the python tests and will be reverted later?

Reply:
Yes, but do we not need to do this eventually if we want to bring this to prod?

Reply:
Eventually yes. But only after we have migrated the consumers in prod. Doing it now before we have tested or migrated to Rust means the Python consumer would be calling the rust processor in prod today, and this path has never really been tried in production at all.

Reply:
Make sense, will remove it once all the tests pass!

Reply:
this change needs to be reverted

Reply:
Done